src/applications/claims-status/sass/claims-status.scss

```diff
@@ -748,8 +746,8 @@ h1:focus {
       border-left-color: transparent;
 
       &::before {
-        background-color: $color-inactive;
-        color: $color-inactive;
+        background-color: $color-gray-light;
+        color: $color-gray-light;
```

Comment:
![image](https://user-images.githubusercontent.com/2008881/119416876-71392f00-bca9-11eb-97e5-2bc63c29f42c.png)
![image](https://user-images.githubusercontent.com/2008881/119416899-831ad200-bca9-11eb-99f6-43d08a2273ba.png)


`$color-gray-light` is `#aeb0b5`: https://design.va.gov/design/color-palette